### ISSUE TITLE
Create a route for submitting that API Data is "Incorrect" or "Out of Date"

### ISSUE THREAD

**Issue by TheScree** [MalekaiProject/malekaiAPI#4]
RethinkDB DB was created and named "issuesQueue". Please utilize this proposed DB entry for creating and submitting issues to this database;

let dataIssue = { 
error_source: 'crowfall.wiki-string', //source should correspond to where the error is coming from
error_user: 'user string/id of person who found error', //use logged in info if user is logged in!
error_ip: 'ipaddress-string', //attempt to grab the IP of person submitting an error if done anonymously/for rate limiting
error_message: 'users can submit brief message if they chose to (and interface allows it)-string',
error_date: now(),
data_type: 'discipline', (or 'power' or 'class' or 'race' or 'recipe' or 'tradeskill', etc.)
data_id: 'id string of the invalid data'
}

**Comment by TheScree** [comment]
Updated this post to reflect the format and database table name for issues to be added to. Working on code now to properly notify us when an updated entry occurs. Testing should be fairly easy. Optional attributes here depending on the source of the submission (can submit via website, or discord bot perhaps?)

**Comment by MichaelFedora** [comment]
```typescript
interface DataIssue {
  error_source: string; // i.e. 'crowfall.wiki', source should correspond to where the error is coming from
  error_user: string; // user string/id of person who found error; use logged in info if user is logged in!
  error_ip: string; // attempt to grab the IP of person submitting an error if done anonymously/for rate limiting
  error_message: string; // users can submit brief message if they chose to (and interface allows it)
  error_date: Date; // "now"
  data_type: string; // ('discipline' or 'power' or 'class' or 'race' or 'recipe' or 'tradeskill', etc.)
  data_id: string; // id of the data
}
```

**Comment by MichaelFedora** [comment]
implemented via df7414e
